olp-cpp-sdk-core/CMakeLists.txt

```diff
@@ -96,6 +97,10 @@ set(OLP_SDK_HTTP_HEADERS
     ./include/olp/core/http/NetworkUtils.h
 )
 
+set(OLP_SDK_MODEL_HEADERS
+    ./include/olp/core/model/Api.h
```

Comment:
Place this under `client/model/` please. Keep it close to the ApiClientLookup.
I would even consider not naming it model at all as it is not auto-generated anymore.